### PR TITLE
Add @mrmorris vacation spot

### PR DESCRIPTION
**Pull request by mrmorris**
Testing local-only PR creation

### CODE REVIEW

File: bios/mrmorris.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
-Ryan Morris
+# Ryan Morris
 
 * A human
 * Not a dog :dog:
 * Has fingers :hand:
+
+## Favorite Vacation Spot
+
+* Singapore
```
Comment (githubteacher) [inline bios/mrmorris.md:9]: I :heart: Singapore!

Comment (githubteacher) [inline bios/mrmorris.md:9]: What if you included the country too?  Is there more than one Singapore?  I am terrible with geography! :see_no_evil:

Comment (mrmorris) [inline bios/mrmorris.md:9]: City-state! It is it's own grandpa. I mean, country.

**Comment by DanielSiefert** [comment]
:+1:

**Pull request merged**